client/src/modules/reports/generate/collectionCapacity/collectionCapacity.config.js

```diff
@@ -0,0 +1,65 @@
+angular.module('bhima.controllers')
+  .controller('collectionCapacityController', CollectionCapacityController);
+
+CollectionCapacityController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService',
+  'AppCache', 'reportData', '$state',
+];
+
+function CollectionCapacityController($sce, Notify, SavedReports, AppCache, reportData, $state) {
+  const vm = this;
+  const cache = new AppCache('collectionCapacity');
+  const reportUrl = 'reports/finance/collectionCapacity';
+
+  vm.previewGenerated = false;
+  vm.reportDetails = {};
+
+  vm.clearPreview = function clearPreview() {
+    vm.previewGenerated = false;
+    vm.previewResult = null;
+  };
+
+  vm.onSelectCashbox = function onSelectCashbox(cashbox) {
```

Comment:
Would it be possible to just use all cashboxes?  Specifically, all `is_auxiliary` cashboxes.  I think that will make the analysis more complete and more useful.